src/commands/components/dev/run.ts

```diff
@@ -87,15 +102,21 @@ export default class RunCommand extends Command {
       ({ requiredConfigVariable: { key } }) => key === connectionKey,
     );
     if (!connection) {
-      ux.error("Failed to find active connection.", { exit: 1 });
+      ux.error("Failed to find active connection with that name.", { exit: 1 });
     }
 
-    const { meta, inputs } = connection;
+    const { meta, inputs, requiredConfigVariable } = connection;
 
-    const fields = inputs.nodes.reduce<Record<string, unknown>>(
-      (result, { name, value }) => ({ ...result, [name]: value }),
-      {},
-    );
+    // Combine templated connection field values with the test instance's field values
+    const fields = {
```

Comment:
Clarification question here -- this may be an edge case or not applicable at all, so just LMK if that's the case.

Is it possible for `connectionTemplate` inputs and regular inputs to share keys/names? And if so, do we care about how merge conflicts are resolved?

The reason I ask is b/c if a `connectionTemplate` input and regular input share a key here (`someSharedKey`), this merge results in the regular inputs winning out. If for whatever reason `inputs.nodes.someSharedKey` comes back with an `undefined` value in the query response (as opposed to being omitted), that results in `someSharedKey` being undefined in the result, regardless of what's in the template.

Reply:
That's a great question. With the way the API is currently implemented, an input value will either be contained within the connection template, or within the instance's values, but not both. So, I don't believe we'll have merge conflicts.